src/doc/trpl/comments.md

```diff
@@ -1,47 +1,45 @@
 % Comments
 
-Now that we have some functions, it's a good idea to learn about comments.
+Now that we have some functions, it’s a good idea to learn about comments.
```

Comment:
I would suggest using [smartypants](https://daringfireball.net/projects/smartypants/), which was designed to do this sort of thing, rather than using untypeable(?) characters in the markdown, especially for common punctuation. The version of [hoedown rustdoc uses supports it](https://github.com/rust-lang/hoedown/blob/238c4d57cce10d33b05cf52a91fc62a09f31ffbb/src/html_smartypants.c), so it'd be a matter of adding an extra pass to rustdoc for the book.